lib/features/ets/quick-link/quick_links_view.dart

```diff
@@ -61,7 +61,7 @@ class _QuickLinksViewState extends State<QuickLinksView>
         children: [
           Expanded(
             child: _buildReorderableGridView(
-                model, model.quickLinkList, _buildDeleteButton),
+                model, model.quickLinkList, _buildDeleteButton, false),
```

Comment:
Pourrais-tu ajouter le nom du paramètre pour éviter d'avoir un booléen magique ?
```suggestion
                model, model.quickLinkList, _buildDeleteButton, blockReorder: false),
```